src/datasources/collectionVariableDrafts.js

```diff
@@ -0,0 +1,66 @@
+import { GraphQLError } from 'graphql'
+import {
+  InvocationType,
+  InvokeCommand,
+  LambdaClient,
+  LogType
+} from '@aws-sdk/client-lambda'
+
+import camelCase from 'lodash/camelCase'
+import mapKeys from 'lodash/mapKeys'
+
+import { downcaseKeys } from '../utils/downcaseKeys'
+import { getLambdaConfig } from '../utils/aws/getLambdaConfig'
+import { parseError } from '../utils/parseError'
+
+export default async (params, context) => {
+  const { headers } = context
+
+  const { authorization: authorizationHeader = '' } = downcaseKeys(headers)
+
+  // Split the token by a space (Bearer xxx.xxx.xxx) and take the
+  // last element to get just the token without the 'Bearer' identifier
+  const token = authorizationHeader.split(' ').pop()
```

Comment:
Slightly convoluted note (I think @william-valencia missed this discussion in Slack)...

We've ended up with a little bit of a mismatch within `earthdata-varinfo`. The functionality to publish CMR records just assumes we have a full `Authorization` header and passes it through, whereas the functionality to search for CMR and download the granule is being given a token (and assumes it is a bearer token).

To simplify things in DAS-1895, I'll be updating the searching/downloading functionality to pass the full `Authorization` header through, without caring too much about what it actually looks like. When that happens, we can simplify this bit by not having to pull out the token header, and just pass it all through.

(I guess this is more of a heads-up for the next PR)

Reply:
TL;DR: No changes in this PR, but probably changes need in the next one.